extensions/oplogPopulator/OplogPopulatorConfigValidator.js

```diff
@@ -1,11 +1,13 @@
 const joi = require('joi');
+const { probeServerJoi } = require('../../lib/config/configItems.joi.js');
```

Comment:
We can let node work out the extension.

```suggestion
const { probeServerJoi } = require('../../lib/config/configItems.joi');
```